src/feature/controls_panel/views/control-component/ControlComponent.tsx

```diff
@@ -361,12 +375,33 @@ const ControlComponent = ({
             placeholder="Enter a number"
             className={clsx(classes.ctrlNumericInput, "border-color")}
             onChange={(e) => {
-              handleCtrlValueChange(setNumberInput, e.target.value);
+              handleCtrlValueChange(
+                setNumberInput,
+                e.target.value,
+                ParamValueType.float
+              );
             }}
             value={currentInputValue || numberInput || 0}
           />
         </Box>
       )}
+      {ctrlObj.name === ControlNames.ArrayNumbericInput && (
+        <Box className={classes.inputBody}>
+          <input
+            type="string"
```

Comment:
Should be `text` for `string` values IMO

Reply:
This is doable but the types in manifests-latest.json are "string" instead of "text". Should I change them as well?